### PR TITLE
Designed the homepage

### PR DESCRIPTION
**Pull request by initishh**
Added a video to the homepage and redesigned the navbar.

### CODE REVIEW

File: src/containers/indexContainer.js [hunk 1]
```diff
@@ -22,8 +41,8 @@ const hrStyle = {
 };
 
 const innerPara = {
-  fontSize: `16px`,
-  fontFamily: `Lato`,
+  fontSize: `20px`,
```
Comment (agathver) [inline src/containers/indexContainer.js]: Prefer using relative units. Consider 16px = 1rem => 20px = 1.25rem.

File: src/components/sitemetadata.js [hunk 2]
```diff
@@ -5,6 +5,10 @@ const Sitemetadata = () => {
   return (
     <Helmet>
       <html lang="en" />
+      <link
+        href="https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700"
```
[the remark this answers is not in the document]
Comment (initishh) [reply src/components/sitemetadata.js]: font weights removed.

File: src/components/sitemetadata.js [hunk 3]
```diff
@@ -5,6 +5,10 @@ const Sitemetadata = () => {
...
+        rel="stylesheet"
+      />
       <link
         href="https://fonts.googleapis.com/css?family=Lato"
```
[the remark this answers is not in the document]
Comment (initishh) [reply src/components/sitemetadata.js]: I am not able to merge css links. I need help with this.

File: src/containers/indexContainer.js [hunk 4]
```diff
@@ -3,15 +3,34 @@ import NavigationBar from "../components/Navigationbar";
...
 
 const styles = {
-  width: `100%`
+  width: `100%`,
+  margin: `0`,
+  padding: `0`
+};
+
+const overlayStyles = {
+  position: `absolute`,
```
[the remark this answers is not in the document]
Comment (initishh) [reply src/containers/indexContainer.js]: Using other position values is breaking my navbar positions. Looking for a fix.

File: src/containers/indexContainer.js [hunk 5]
```diff
@@ -3,15 +3,34 @@ import NavigationBar from "../components/Navigationbar";
...
+const containerStyles = {
+  display: `flex`,
+  justifyContent: `center`
 };
 
 const headingText = {
-  fontFamily: `Courier New`,
-  fontSize: `36px`,
-  marginTop: `5%`
+  fontFamily: `Montserrat`,
```
[the remark this answers is not in the document]
Comment (initishh) [reply src/containers/indexContainer.js]: Just wanted a common font for all the title. Do have a look.

Screenshot from 2019-04-28 23-46-46

File: src/containers/indexContainer.js [hunk 6]
```diff
@@ -3,15 +3,34 @@ import NavigationBar from "../components/Navigationbar";
...
+  fontSize: `40px`,
+  fontWeight: `bold`,
```
[the remark this answers is not in the document]
Comment (initishh) [reply src/containers/indexContainer.js]: Fixed.

File: src/containers/indexContainer.js (hunk 1 above)
Comment (initishh) [reply src/containers/indexContainer.js]: fixed.

**Comment by ganeshpatro321** [comment]
@initishh Please push your commits, so we can review them.

**Comment by initishh** [comment]
@ganeshpatro321 currently working on making the site responsive. The site breaks when the screen size is decreased. I will push the code once it is resolved.

File: src/components/Navigationbar.js [hunk 7]
```diff
@@ -1,20 +1,53 @@
 import React from "react";
 import { Navbar, Nav } from "react-bootstrap";
 import Sitemetadata from "./sitemetadata";
+import "./navbarCSS.css";
+
+const textStyle = {
+  color: `#6d48e5`,
+  fontSize: `20px`,
+  padding: `4px`,
+  fontSize: `20px`,
```
[the remark this answers is not in the document]
Comment (initishh) [reply src/components/Navigationbar.js]: Sorry about that.

**Comment by initishh** [comment]
Do we have any Zairza logo? We can use it here.

**Comment by initishh** [comment]
I used navbar color is in accordance with the video. 
Do suggest other colors and we can have a check.

img

File: src/components/sitemetadata.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (initishh) [reply src/components/sitemetadata.js]: CSS merged.

**Comment by ankitjena** [comment]
@initishh  Great work!. Just some issues. The "WONDER THINK CREATE" breaks to two line. Try to make it stick to a single line or three line when small. This looks weird. Also reduce the fontsize of the text below to match the underlying card components. Looks too big on small screen.

**Pull request closed without merging**